etc/plans/Attack/AlignAndPassRapid.beh

```diff
@@ -11,7 +11,7 @@
     <parameters id="1440849496542" name="MISSING_NAME" comment="" key="TeamMatePlanName0" value="AttackPlay"/>
     <parameters id="1440849505103" name="MISSING_NAME" comment="" key="TeamMatePlanName1" value="AttackPlay"/>
     <parameters id="1440849514834" name="MISSING_NAME" comment="" key="TeamMateTaskName0" value="ReceivePassInGame"/>
-    <parameters id="1440849527880" name="MISSING_NAME" comment="" key="TeamMateTaskName1" value="SupportAttack"/>
+    <parameters id="1440849527880" name="MISSING_NAME" comment="" key="TeamMateTaskName1" value="Block"/>
   </configurations>
```

Comment:
The "SupportAttack" Task doen't exist in our current Task Repository anymore. The 2nd possible pass receiver has the Task "Block".